src/tmp/Factories/UniversalReceiverAddressStoreInit__factory.ts

```diff
@@ -0,0 +1,245 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
why in /tmp/ folder?

What about a `/generated/` folder?

Reply:
It's a copy/pasted file, once the `universalprofile-smart-contracts` package contains the init versions, the folder will disappear.